api/client/client.go

```diff
@@ -45,11 +44,20 @@ type Remote interface {
 func NewServer(addr string) Remote {
 	addrs := strings.Split(addr, ",")
 
+	for i := range addrs {
```

Comment:
put normalization in its own function and thanks for the suggestions @kisom, I took the strings route.